VRDR/DeathRecord.cs

```diff
@@ -1382,7 +1382,11 @@ public string CertificationRoleHelper
             }
             set
             {
-                if ((value != null) && !VRDR.Mappings.CertifierTypes.FHIRToIJE.ContainsKey(value))
+                if (value == null || String.IsNullOrEmpty(value.Trim())){
```

Comment:
Can use `String.IsNullOrWhitespace` instead of combining `Trim` and `IsNullOrEmpty`

Reply:
modified as per suggestion...thx